sphinx-docs/developer_docs/logging.rst

```diff
@@ -0,0 +1,13 @@
+Logging
+=======
+
+If you wish to view output from the server, you have a few options:
+
+*  Start the server using the command ``kalite manage kaserve --production``.
+   This will start the server using CherryPy and a single thread, with output going to stdout and stderr.

```

Comment:
What I would suggest would rather be `kalite start --foreground` or `kalite manage runserver` and then not mention `kalite manage kaserve`.